app-doc/gimp-help/gimp-help-2.10.34-r2.ebuild

```diff
@@ -13,7 +13,8 @@ SRC_URI="mirror://gimp/help/${P}.tar.bz2"
 LICENSE="FDL-1.2+"
 SLOT="2"
 KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
-IUSE="nls"
+IUSE="nls test"
+RESTRICT="!test? ( test )"
```

Comment:
Why the flag and the RESTRICT? I don't think they are needed here.

Reply:
Removed